README.md

```diff
@@ -2,25 +2,25 @@
 
 ## Contents
 
--   [Introduction](#introduction)
--   [Free for Individual Developers, Paid for Enterprise](#free-for-individual-developers-paid-for-enterprise)
--   [Key Features](#key-features)
--   [Videos](#videos)
--   [Download](#download)
--   [Supported Distros](#supported-distros)
--   [Host Requirements](#host-requirements)
--   [Installing Sysbox](#installing-sysbox)
--   [Using Sysbox](#using-sysbox)
--   [Documentation](#documentation)
--   [Integration with Container Managers](#integration-with-container-managers)
--   [Troubleshooting](#troubleshooting)
--   [Filing Issues](#filing-issues)
--   [Support](#support)
--   [We want your feedback](#we-want-your-feedback)
--   [Uninstallation](#uninstallation)
--   [About Nestybox](#about-nestybox)
--   [Contact](#contact)
--   [Thank You](#thank-you)
+- [Introduction](#introduction)
+- [Free for Individual Developers, Paid for Enterprise](#free-for-individual-developers-paid-for-enterprise)
+- [Key Features](#key-features)
```

Comment:
The TOC needs updating, does not match the headlines in the text

(No "Free for individual developers", "Key Features -> Features", etc)

Reply:
Mmm, was expecting Markdown linter to catch this, I guess we can't trust it that much ...

Done.